sklearn/lda.py

```diff
@@ -180,6 +180,12 @@ class LDA(BaseEstimator, LinearClassifierMixin, TransformerMixin):
     covariance_ : array-like, shape (n_features, n_features)
         Covariance matrix (shared by all classes).
 
+    explained_variance_ratio_ : array, (n_components)
```

Comment:
you forgot the ,

(n_components) -> (n_components,)

it's a tuple of length 1